README.md

```diff
@@ -1,6 +1,8 @@
-# Globant Java Bootcamp 2018
+﻿# Globant Java Bootcamp 2018
 ## Spring - REST - Git - Maven - Testing - RDB
 
+Group-1: Raverta, Claudio Marco (claudiorave). Perez, Jefferson.
```

Comment:
"Every participant must create its own PR including his/her complete name and nickname."
Que figure solo tu nombre. Y Jefferson debería hacer su propio PR

Reply:
Chequear que el PR de giorgio ya fue mergeado. Incluir esto bajo su comentario